package/MDAnalysis/make_bundle.py

```diff
@@ -52,39 +54,45 @@ def make_bundle(*args, **kwargs):
     # TODO - use Group
     # TODO - careful about overwriting existing sims/category values
 
-    names = kwargs.get('names', map(str, range(len(args[0]))))
-    if len(args) == 2:
+    if not names:
```

Comment:
Correct check. `if names is not None`. There are more expressions that can evaluate to false in python